Bottom: 90a0551dae655bc5f32f2c86eb6633795d6d8051
Top:    e244c4b791084bbae0b78207d1fd633d23db0cc8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-08 02:05:42 +0000

Not so sure about these MyStrdup calls


---

diff --git a/src/create.c b/src/create.c
index d92e6e8..9fb305b 100644
--- a/src/create.c
+++ b/src/create.c
@@ -587,7 +587,7 @@ CreateNewText (LayerTypePtr Layer, FontTypePtr PCBFont,
   text->Direction = Direction;
   text->Flags = Flags;
   text->Scale = Scale;
-  text->TextString = MyStrdup (TextString, "CreateNewText()");
+  text->TextString = strdup (TextString);
 
   /* calculate size of the bounding box */
   SetTextBoundingBox (PCBFont, text);
@@ -893,7 +893,7 @@ AddTextToElement (TextTypePtr Text, FontTypePtr PCBFont,
   Text->Direction = Direction;
   Text->Flags = Flags;
   Text->Scale = Scale;
-  Text->TextString = (TextString && *TextString) ? strdup (TextString) : NULL;
+  Text->TextString = strdup (TextString);
 
   /* calculate size of the bounding box */
   SetTextBoundingBox (PCBFont, Text);
